keyboards/zeix/qwertyqop60hs/info.json

```diff
@@ -0,0 +1,96 @@
+{
+    "keyboard_name": "QQ 60% Hotswap",
+    "maintainer": "itsme-zeix",
+    "manufacturer": "qwertyqop",
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "usb": {
+        "vid": "0x4C27",
+        "pid": "0x2900",
+        "device_version": "0.0.1"
+    },
+    "features": {
+        "bootmagic": true,
+        "mousekey": true,
+        "extrakey": true,
+        "console": false,
+        "command": false,
+        "nkro": true
+    },
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "rows": ["GP6","GP5","GP4","GP0","GP12"],
+        "cols": ["GP28","GP27","GP26","GP25","GP24","GP23","GP22","GP3","GP2","GP1","GP8","GP9","GP10","GP11","GP7"]
+    },
+    "layouts": {
+        "LAYOUT": {
```

Comment:
looking at the image in `readme.md`, could the following Community Layouts (`layouts/default/`) be added:
- 60_ansi
- 60_ansi_split_bs_rshift
- 60_ansi_tsangan
- 60_tsangan_hhkb
- 60_hhkb

or, a `matrix_diagram.md` provided?

Reply:
got it, I'll work on it and include it later!